components/adv-post-form.js

```diff
@@ -17,14 +17,34 @@ export function AdvPostInitial ({ forward, boost }) {
   }
 }
 
-export default function AdvPostForm () {
+export default function AdvPostForm ({ edit, isLink }) {
   const me = useMe()
 
   return (
     <AccordianItem
       header={<div style={{ fontWeight: 'bold', fontSize: '92%' }}>options</div>}
       body={
         <>
+          {!edit && isLink &&
+            <MarkdownInput
+              label={
+                <div className='d-flex align-items-center'>comment
```

Comment:
If you go down the `item.text` path, maybe this should be renamed to `text`? Then it's consistent with the other form fields since their names match exactly the respective database column.

But maybe "comment" is still a better user description than "text".

Reply:
I went ahead and changed it to "text" since it looks different to the user than a comment. (If there's any doubt, the info button is there to clarify any doubt.)